examples/pytorch/dgi/README.md

```diff
@@ -0,0 +1,38 @@
+Deep Graph Infomax (DGI)
+========================
+
+- Paper link: [https://arxiv.org/abs/1809.10341](https://arxiv.org/abs/1809.10341)
+- Author's code repo (in Pytorch):
+  [https://github.com/PetarV-/DGI](https://github.com/PetarV-/DGI)
+
+Dependencies
+------------
+- PyTorch 0.4.1+
+- requests
+
+```bash
+pip install torch requests
+```
+
+How to run
+----------
+
+Run with following:
+
+```bash
+python train.py --dataset=cora --gpu=0 --self-loop
+```
+
+```bash
+python train.py --dataset=citeseer --gpu=0
+```
+
+```bash
+python train.py --dataset=pubmed --gpu=0
+```
+
+Results
+-------
+* cora: ~81.6 (81.2-82.1) (paper: 82.3)
+* citeseer: ~69.4 (paper: 71.8)
```

Comment:
This number seems to be lower than dgi paper. I'm trying to find the difference between your implementation and dgi official implementation.

Reply:
The citeseer's performance is much lower than dgi paper. I was trying to figure out, but not found the reason.

Reply:
I did 5 recent experiments of citeseer. The result is between 69.3 and 71.8.
The detailed results are list below:
70.3
71.3
70.8
71.8
69.3

Reply:
I see, I've ran your code too and yield similar results. I also tried different initializers as Xavier in your Discriminator as the author did, but that did not influence the result.